### PR TITLE
Migrate to GHA

### PR DESCRIPTION
**Pull request by p-galligan**
Migrates CI/CD from Travis to GHA

Also updates the RAC stylesheet to the most recent version.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      uses: actions/setup-node@v4.0.3
+      with:
+        node-version: 'lts/*'
+        cache: 'yarn'
+
+    - name: Install dependencies
+      run: yarn install
+
+    - name: Build project
+      run: CI=false yarn build
```
Comment (helrond) [inline .github/workflows/tests.yml]: I think this step actually needs to be in the deploy (this creates a built site to be deployed)

Comment (helrond) [reply .github/workflows/tests.yml]: You may also need the preceding step (yarn install) in that file as well. I believe it needs to be both places.

**Review by helrond — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (p-galligan) [reply .github/workflows/tests.yml]: Ahhhh, I am not familiar with react. That makes sense. I think I also have to install node first (based on some googling), so I've added that as well.

**Review by HaSistrunk — APPROVED** [review]
I _think_ this is good, but I'm not 100% on the specifics of this one. (I also piggbacked a small update to the style library version, if that's cool)

File: .github/workflows/tests.yml (hunk 1 above)
Comment (helrond) [reply .github/workflows/tests.yml]: I'm still not seeing a build in the deploy file. I'm also 99% sure you don't need to build here in order to run tests.

Comment (p-galligan) [reply .github/workflows/tests.yml]: Got that build section in the deploys and I've removed it from tests.

**Review by helrond — APPROVED** [review]
I believe this is correct!

**Pull request merged**